informer/disk/disk.go

```diff
@@ -85,10 +103,10 @@ func (disk *Informer) GetMetric(ctx context.Context) *api.Metric {
 	disk.mu.Unlock()
 
 	if rpcClient == nil {
-		return &api.Metric{
+		return []*api.Metric{&api.Metric{
```

Comment:
Metric is expired. How did the pusher handle this before without resetting the timer to 0?

Reply:
A bug probably?